WordPress/src/main/java/org/wordpress/android/util/SiteUtils.java

```diff
@@ -35,8 +36,44 @@
      * -- 12.9 OPTED OUT (were auto-opted in but turned it OFF) -> turn all sites OFF in 13.0
      *
      */
-    public static void migrateAppWideMobileEditorPreferenceToRemote(final Context context,
+    public static void migrateAppWideMobileEditorPreferenceToRemote(final AccountStore accountStore,
+                                                                    final SiteStore siteStore,
                                                                     final Dispatcher dispatcher) {
+        // In a later version we might override mobile_editor setting if it's set to `aztec` and show a specific notice
+        // for these users ("We made a lot of progress on the block editor and we think it's now better than
+        // the classic editor, we switched it on, but you can change the configuration in your Site Settings").
+        // ^ This code should be here.
+
+        // If the user is already in the rollout group, we can skip this the migration.
+        if (AppPrefs.isUserInGutenbergRolloutGroup()) {
+            return;
+        }
+
+        // Check if the user has been "randomly" selected to enter the rollout group.
+        //
+        // For self hosted sites, there are often one or two users, and the user id is probably 0, 1 in these cases.
+        // If we exclude low ids, we won't get an not an homogeneous distribution over self hosted and WordPress.com
+        // users, but the purpose of this is to do a progressive rollout, not an necessarily an homogeneous rollout.
+        //
+        // To exclude ids 0 and 1, to rollout for 10% users,
+        // we'll use a test like `id % 100 >= 90` instead of `id % 100 < 10`.
+        if (accountStore.getAccount().getUserId() % 100 >= (100 - GB_ROLLOUT_PERCENTAGE)) {
```

Comment:
Not sure i've got the comment about self hosted sites correctly. 
It seems to me that a lot of the self hosted installations won't be included in the roll out group until we raise the bar to 100, or something very close to that value, since most of those sites do only have one or two users, with their IDs be `0` or `1`. 
Is that correct? If yes, we're promoting GB-mobile more on the .com side, and less on the .org site, with a spike of .org sites that will be included only at the end of the rollout phase.

Reply:
👍